express/src/routes/sign-in.ts

```diff
@@ -52,5 +53,6 @@ router.get("/no-account", (req, res) => {
 
 router.get("/forgot-password", forgotPasswordForm);
 router.get("/check-email-password-reset", checkEmailPasswordReset);
+router.post("/check-email-password-reset", resendForgotPassword);
```

Comment:
I believe `emailIsPresentInSession ` should be used for these routes (check with @JohnWattsGDS )

Reply:
Done.